src/lib/govuk/FormElement.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  // The label to show
+  export let label: string;
+  // A unique (per page) ID of the form element being labelled. Something in
```

Comment:
I wish we didn't have to make this up at all. Previously we've been doing `<label>xyz <input ... /></label>`, but this breaks the govuk styling by eliminating a line-break. So, this Svelte component slightly helps.